spec/features/service_providers_spec.rb

```diff
@@ -197,6 +197,18 @@
       click_on t('forms.buttons.trigger_idp_refresh')
       expect(page).to have_content(I18n.t('notices.service_providers_refreshed'))
     end
+
```

Comment:
I suggest we add a feature test to insure the checkbox does not display for non admins

Reply:
Done!